agent/bench-scripts/pbench-dbench

```diff
@@ -20,10 +20,18 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 
 benchmark_rpm=$script_name
 benchmark="dbench"
-if [[ -z "$benchmark_bin" ]]; then
-	benchmark_bin=/usr/local/bin/$benchmark
+if [[ -z "${benchmark_bin}" ]]; then
+	benchmark_bin="$(resolve_benchmark_bin "${benchmark}")"
+	if [[ -z "${benchmark_bin}" ]]; then
+		error_log "[${script_name}] ${benchmark} executable not found on PATH=${PATH}"
+		exit 1
+	fi
+fi
+ver="$(getconf.py version ${benchmark})"
+if [[ -z "${ver}" ]]; then
+	error_log "${script_name}: package version is missing in config file"
+	exit 1
 fi
```

Comment:
While there's a quick reference to "version" in the git log message, it seems this should be highlighted a bit more clearly.

Also, given the generalization of the messages in other wrappers, it almost seems these were written to be factored into a common `base` function rather than duplicated across all the scripts; is that worth pursuing here?

And finally (and I hate to even say this), the fact that this doesn't have any impact on legacy gold files makes me suspect there's something missing. ☹️

Reply:
Rather than replicating this code through all the scripts, I'm wondering whether it wouldn't be feasible to attach it to `check_install_rpm` (or wherever it's actually used) instead.  (In at least one case, I think that this required configuration doesn't actually get used unless the `--install` switch is provided....)

Reply:
This is back-port of the changes from PR #1719, as stated in the commit message and in the PR description.

Are these comments stemming from the back-port being incorrect as compared to the original code in `main`?  If so, great.  If not, then we should fix this is `main` first.

Reply:
oops; I completely overlooked that in the stream of PRs, so I'll retract my comments. Let's make this consistent with main. The comments would probably be useful going forward on main, but I don't see much priority in worrying about them given everything else going on.